map.go

```diff
@@ -498,10 +520,23 @@ func (m *Map) Iterate() *MapIterator {
 	return newMapIterator(m, nil)
 }
 
+// randomizeKey() is here for kernel versions < 4.4.132, which don't support using
+// a NULL `key` with the `BPF_MAP_GET_NEXT_KEY` command to the bpf syscall.
+func (m *Map) randomizeKey() []byte {
+	m.randKey = make([]byte, int(m.abi.KeySize))
+	if m.rand == nil {
+		m.rand = rand.New(rand.NewSource(int64(os.Getpid())))
```

Comment:
Is the source intentionally the same for multiple maps, so they produce the same sequence of keys?

Reply:
I forgot to update the source thanks